.eslintignore

```diff
@@ -5,10 +5,5 @@ flow-typed/**
 packages/*/build/**
 packages/*/build-es5/**
 types/**
-website/build
-website/node_modules
-website/core/metadata*.js
-website/i18n/*.js
-website/src/jest/docs
-website/src/jest/blog
+website/**
```

Comment:
I still don't think this is necessary

Reply:
Thanks @SimenB I sent this PR so I could make edits. Will review.